### ISSUE TITLE
IllegalArgumentException The datetime zone id ‘Europe/Vienna’ is not recognised

### ISSUE THREAD

**Issue by majzelm** [dlew/joda-time-android#284]
I have a problem with latest two versions of library **2.10.9** and **2.10.9.1** together with the use of DexGuard. So far, everything has worked reliably with all versions (latest working **2.10.7.2**). Now, app is crashing when `DateTimeZone.forID("Europe/Vienna")` or `DateTimeZone.forTimeZone(TimeZone.getTimeZone("Europe/Vienna"))` is called.
```
java.lang.IllegalArgumentException: The datetime zone id 'Europe/Vienna' is not recognised
        at org.joda.time.DateTimeZone.RemoteActionCompatParcelizer(Unknown Source:398)
```

The problem only occurs with the production version of app (when DexGuard is enabled). With developer build everything works. Does anyone have a similar problem?

**Comment by dlew** [comment]
Does this error happen for any timezone, or specifically just `Europe/Vienna`?

Regardless, I haven't used DexGuard before so I don't know what could be happening, short of DexGuard removing require assets included by this library. I assume the reason the developer build works is because DexGuard isn't being run. [We've got proguard rules that protect against required assets being removed](https://github.com/dlew/joda-time-android/blob/main/library/proguard-rules.txt), but I don't know if DexGuard obeys those rules.

**Comment by kmayoral** [comment]
Hi @majzelm , sorry for the late reply here but could it be that the library was not initializing due to your app having either multiple processes or you having a custom implementation of the AndroidX app startup library? 

I had this issue at first when updating to the latest version of the library only to realize that the previous `JodaTimeAndroid.init()` call does nothing now. In normal situations, the app startup library integration that was added would have automatically initialized the library but our app disabled auto initialization of components through app-startup and so I had to call `AppInitializer.getInstance(this).initializeComponent(JodaTimeInitializer::class.java)` instead (see updated [docs](https://github.com/dlew/joda-time-android#multi-process-apps)).

Anyways, just figured I'd post a comment here in case anyone ran into this in the future (although it's well documented on the main readme, I just didn't see it at first).
